ddtrace/opentracer/tracer.py

```diff
@@ -33,7 +35,8 @@
 class Tracer(opentracing.Tracer):
     """A wrapper providing an OpenTracing API for the Datadog tracer."""
 
-    __slots__ = ['_enabled', '_debug', '_service_name', '_tracer']
+    __slots__ = ['_enabled', '_debug', '_service_name', '_tracer',
```

Comment:
Another `__slots__` we should probably drop since the [base class](https://github.com/opentracing/opentracing-python/blob/2.0.0rc2/opentracing/tracer.py#L32) doesn't use them.